genesyscloud/external_contacts/resource_genesyscloud_externalcontacts_contact_schema.go

```diff
@@ -329,7 +334,7 @@ func ExternalContactExporter() *resourceExporter.ResourceExporter {
 	return &resourceExporter.ResourceExporter{
 		GetResourcesFunc: provider.GetAllWithPooledClient(getAllAuthExternalContacts),
 		RefAttrs: map[string]*resourceExporter.RefAttrSettings{
-			"external_organization": {}, //Need to add this when we external orgs implemented
+			"external_organization_id": {RefType: "genesyscloud_contacts_organization"},
```

Comment:
This should be "genesyscloud_externalcontacts_organization". You could also import the package `external_contacts_organization` and reference the const string `ResourceType` instead of hard-coding this string.